### ISSUE TITLE
Key Details NINO abbreviation

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#479]
## Describe the issue
The definition term for the NINO should read as National Insurance number.
Suggestion from audit: `Should be 1 list and broken up with divs, rather than 2 lists`
